docs/source/theory/supportedEquationSet.rst

```diff
@@ -422,6 +402,104 @@ following temperature form:
 
    q_j = -\kappa \frac{\partial T}{\partial x_j}.
 
+ABL Forcing Source Terms
+++++++++++++++++++++++++
+
+In LES of wind plant atmospheric flows, it is often necessary to
+drive the flow to a predetermined vertical velocity and/or temperature profile.
+In Nalu, this is achieved by adding appropriate
+source terms :math:`\mathrm{f}_i` to the
+momentum equation :eq:`favmom` and
+:math:`S_\theta` to the enthalpy equation :eq:`fav-enth`.
+
+First, the momentum source term is discussed.
+The main objective of this implementation is to force the volume averaged velocity at
+a certain location to a specified value (:math:`<\mathrm{u}_i>=\mathrm{U}_i`).
+The brackets used here, :math:`<>`, mean volume averaging over a certain region.
+In order to achieve this, a source term must be applied to the momentum equation.
+This source term can be better understood as a proportional controller within the
+momentum equation.
+
+The velocity and density fields can be decomposed into a volume averaged component
+and fluctuations about that volume average as
+:math:`\mathrm{u}_i = \left< \mathrm{u}_i \right> + \mathrm{u}_i'` and
+:math:`\bar{\rho} = \left< \bar{\rho} \right> + \bar{\rho}'`.
+A decomposition of the plane averaged momentum at a given instance in time is then
+
+.. math::
+       \left< \bar{\rho}  \mathrm{u}_i  \right>  =
+        \left< \bar{\rho} \right> \left< \mathrm{u}_i \right>
+        + \left< \bar{\rho}'  \mathrm{u}'_i  \right>.
+
+We now wish to apply a momentum source based on a desired spatial averaged velocity
+:math:`\mathrm{U}_i`.
+This can be expressed as:
+
+.. math::
+       \left< \bar{\rho}  \mathrm{u}_i^*  \right>  =
+        \left< \bar{\rho} \right> \left< \mathrm{u}^*_i \right>
+        + \left< \bar{\rho}'  {\mathrm{u}^*_i}'  \right>,
+
+where :math:`\mathrm{u}_i^*` is an unknown reference velocity field whose volume
+average is the desired  velocity :math:`\left< \mathrm{u}_i^* \right> = \mathrm{U}_i`.
+Since the correlation :math:`\left< \bar{\rho}'  \mathrm{u^*}'_i  \right>`
+is unknown, we assume that
+
+.. math::
+    \left< \bar{\rho}'  \mathrm{u^*}'_i  \right>
+    =
+    \left< \bar{\rho}'  \mathrm{u}'_i  \right>
+
+such that the momentum source can now be defined as:
+
+.. math::
+   :label: abl-mom-source
+
+   {\mathrm{f}_i} = \alpha_u
+        \left(  \, \frac{\left< \bar{\rho} \right> \mathrm{U_i}
+        - \left< \bar{\rho} \right> \left< \mathrm{u}_i \right>}
+        {\Delta t}\right)
+
+where :math:`\left< \right>` denotes volume averaging at a
+certain time :math:`t`,
+:math:`\mathrm{U}_i` is the desired spatial averaged
+velocity,
+and :math:`\Delta t` is the time-scale between when the source term is computed
+(time :math:`t`) and when it is applied (time :math:`t + \Delta t`).
+This is typically chosen to be the simulation time-step.
+In the case of an ABL simulation with flat terrain, the voulme averaging is done
+over an infinitesimal small plane over the :math:`x` and :math:`y` directions,
```

Comment:
"over an infinitesimally thin slice in the x and y directions..."

Reply:
Hey @mbarone81, Thanks, I have corrected this.